sitemedia/js/components/books/BooksSearch.js

```diff
@@ -0,0 +1,166 @@
+                    >
+                    </search-facet>
+                </div>
+            </sui-segment>
+            <sui-segment inverted class="active-items">
+                <label>Selected</label>
+                <div class="active-facet-choice" v-for="(choice, index) in activeFacetChoices" :key="index">
+                    <sui-label>
+                        <span class="facet-name">{{ facetConfig.find(facet => facet.name === choice.facet).label }}: </span>
+                        <span class="choice-name">{{ choice.value }}</span>
+                        <sui-icon name="delete" @click="toggleFacetChoice(choice)" />
+                    </sui-label>
+                </div>
+            </sui-segment>
+            <sui-segment inverted>
+                <search-sort></search-sort>
+            </sui-segment>
+        </form>
+    </div>`,
+    components: {
+        SearchFacet,
+        SearchFilter,
+        SearchSort,
+    },
+    data() {
+        return {
+            tabs: [
+                ['author', 'editor', 'translator'],
+                ['publisher', 'pub_year'],
+                ['language', 'subject'],
+                ['annotator']
+            ],
+            activeTab: 0,
+            facetConfig: [
+                {
+                    name: 'author',
+                    label: 'Author',
+                    type: 'text',
+                    search: true,
+                    width: 6,
+                },
+                {
+                    name: 'editor',
+                    label: 'Editor',
+                    type: 'text',
+                },
+                {
+                    name: 'translator',
+                    label: 'Translator',
+                    type: 'text',
+                },
+                {
+                    name: 'publisher',
+                    label: 'Publisher',
+                    type: 'text',
+                    search: true,
+                    width: 6
+                },
+                {
+                    name: 'pub_year',
+                    label: 'Publication Year',
+                    type: 'range',
+                },
+                {
+                    name: 'language',
+                    label: 'Language',
+                    type: 'text',
+                },
+                {
+                    name: 'subject',
+                    label: 'Subject',
+                    type: 'text',
+                },
+                {
+                    name: 'annotator',
+                    label: 'Annotator',
+                    type: 'text',
+                },
+            ],
+        }
+    },
+    computed: {
+        ...mapState([
+            'filters',
+            'totalResults',
+            'facetChoices',
+        ]),
+        ...mapGetters([
+            'activeFacets',
+            'activeFacetChoices',
+            'formState',
+        ]),
+    },
+    created() {
+        this.setEndpoint('facets/')
```

Comment:
Can we make this configurable too? I'm imagining a few options being set in the template - not sure what makes the most sense. But we should set this via Django URL rather than hard-coding it here.

Reply:
Yep, another thing we can pass in using html.